bin/varnishd/mgt/mgt_jail_unix.c

```diff
@@ -229,17 +229,6 @@ vju_subproc(enum jail_subproc_e jse)
 	} else {
 		AZ(setuid(vju_uid));
 	}
-
-#ifdef __linux__
```

Comment:
Can you please remove this one too: https://github.com/varnishcache/varnish-cache/pull/4141/commits/31ef342958ebc8d62845818b2903d037d9edee07#diff-c66495bf983bac2ecc16d0b634508370d82a479bff35ab7ba642ea8fd18bfa66R47-R49 I don't think it is needed anymore.